tests/handlers/test_receipts.py

```diff
@@ -130,7 +131,6 @@ def test_handles_missing_content_of_m_read(self):
             [
                 {
                     "content": {
-                        "$14356419ggffg114394fHBLK:matrix.org": {ReceiptTypes.READ: {}},
```

Comment:
I'm not sure if this makes sense -- would we ever end up with an input that has no receipts like this?

Reply:
(It seems this either needs to go in both the input and output or be in both?)

Reply:
Arguably, since this wouldn't ever really happen, we don't need a test case for it though I guess this sort of documents the behaviour of the method in the case this would happen. If it did happen, I think it does make sense to do what we do here, since leaving it in isn't defined by the spec, as far as I know